app/controllers/enrollments_email_templates_controller.rb

```diff
@@ -0,0 +1,22 @@
+class EnrollmentsEmailTemplatesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :extract_enrollment
+
+  def index
```

Comment:
I got a few interrogations about the extensibility of this API. 

- This route can be fetch by a user without instructor rights. Is this a problem ?
- In a more general approach, this route send all templates available independently of the user's rights defined here : https://github.com/betagouv/signup-back/blob/1609ce2917d6d8dc92bcaced24fb64d24586cd39/app/policies/enrollment_policy.rb#L14-L32 . This might result in information leak.

Note that the current implementation is ok for a first iteration.

Reply:
oh ! thanks for this testing effort !

The scope do not allow every user to get the templates but it allows owner, and GDPR contacts to see the templates. In my opinion this might result in information leakage. I'd rather « harden » the access to this route only for instuctors indeed.

Reply:
Ok, I will harden this.